crates/cli/runner/src/lib.rs

```diff
@@ -52,17 +52,24 @@ impl CliRunner {
             // after the command has finished or exit signal was received we shutdown the task
             // manager which fires the shutdown signal to all tasks spawned via the task
             // executor and awaiting on tasks spawned with graceful shutdown
-            task_manager.graceful_shutdown_with_timeout(std::time::Duration::from_secs(10));
+            task_manager.graceful_shutdown_with_timeout(Duration::from_secs(5));
```

Comment:
this seems reasonable, none of our graceful tasks should even take 1s to shutdown